api/envoy/config/filter/http/jwt_authn/v2alpha/BUILD

```diff
@@ -11,3 +11,13 @@ api_proto_library_internal(
         "//envoy/api/v2/route",
     ],
 )
+
+api_go_proto_library(
+    name = "jwt_authn",
+    proto = ":jwt_authn",
+    deps = [
+        "//envoy/api/v2/core:base_go_proto",
+        "//envoy/api/v2/core:http_uri_go_proto",
+        "//envoy/api/v2/route_go_proto",
```

Comment:
I think this line isn't quite right, it's probably `//envoy/api/v2/route:route_go_proto`, this is why CI is failing.

Reply:
Ah thank you, I was having a little trouble figuring out the error. Oops!